### PR TITLE
Help button

### PR DESCRIPTION
**Pull request by raoaliahmad**

### CODE REVIEW

File: web/src/modules/Attendance/index.js [hunk 1]
```diff
@@ -122,7 +122,7 @@ class Attendance extends Component {
 
 				return [...agg, {
 					number: student.Phone,
-					text: this.props.attendance_message_template.replace(/\$NAME/g, student.Name).replace(/\$STATUS/g, att.status)
+					text: `Date: ${moment().format("DD/MM/YYYY")}\n${this.props.attendance_message_template.replace(/\$NAME/g, student.Name).replace(/\$STATUS/g, att.status)}`
```
Comment (taimur38) [inline web/src/modules/Attendance/index.js]: checked this with @Ayesha19 ?

File: web/src/modules/Help/index.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+                </div>
+                <div>
+                {
+                  smsOption === "SIM" ? 
+                    <a href={smsIntentLink({
+                      messages: [{ number, text }],
+                      return_link: window.location.href 
+                      })} onClick={this.logSms} className="button blue">Send using Local SIM</a> :
+                  
+                    <div className="button" onClick={() => sendMessage( text, number)}>Can only send using Local SIM</div>
```
Comment (taimur38) [inline web/src/modules/Help/index.js]: is this what we say in other places for non-sim message sending? I thought we showed an alert that does this, not change the button text.

File: web/src/modules/Landing/index.js [hunk 3]
```diff
@@ -120,9 +122,12 @@ class Landing extends Component {
 							<Link to="/settings" className="button red-shadow" style={{backgroundImage: `url(${settingsIcon})` }}>Settings</Link>
 						</div> : false}
 						<div className="row">
-							<Link to="/student?forwardTo=prospective-student" className="button yellow-shadow" style={{backgroundImage: `url(${prospective})` }}>Prospective</Link>
+						{ user.Admin || setupPage ?<Link to="/student?forwardTo=prospective-student" className="button yellow-shadow" style={{backgroundImage: `url(${prospective})` }}>Prospective</Link> : false}
 							<div className="button yellow-shadow" onClick={logout} style={{backgroundImage: `url(${switchUserIcon})` }}>Logout</div>
 						</div>
+						<div className="row">
+							<Link to="/help" className="button yellow-shadow" style={{backgroundImage: `url(${Help})` }}>Help</Link>
```
Comment (taimur38) [inline web/src/modules/Landing/index.js]: why are all these elements getting yellow shadow? should change it up

File: web/src/modules/Attendance/index.js (hunk 1 above)
Comment (raoaliahmad) [reply web/src/modules/Attendance/index.js]: yes

File: web/src/modules/Help/index.js (hunk 2 above)
Comment (raoaliahmad) [reply web/src/modules/Help/index.js]: Yup thats how we do this everywhere

**Pull request merged**